gsuite_reports_rules/gsuite_drive_visibility_change.py

```diff
@@ -168,3 +186,10 @@ def title(event):
         f"[{ALERT_DETAILS[log]['DOC_TITLE']}] externally visible to [{sharing_scope}] with "
         f"[{alert_access_scope}] access"
     )
+
+
+def severity(event):
+    log = event.get("p_row_id")
+    if ALERT_DETAILS[log]["TARGET_DOMAIN"] in PUBLIC_PROVIDERS:
```

Comment:
This is for Gsuite domains I think. You're going to have to chop up each address in TARGET_USER_EMAILS to do what you're trying to do.

Reply:
Good call, corrected